test/functional/apps/management/_import_objects.js

```diff
@@ -189,27 +188,27 @@ export default function({ getService, getPageObjects }) {
 
       it('should import saved objects with index patterns when index patterns does not exists', async () => {
         // First, we need to delete the index pattern
-        await PageObjects.settings.navigateTo();
-        await PageObjects.settings.clickKibanaIndexPatterns();
-        await PageObjects.settings.removeLogstashIndexPatternIfExist();
+        const elements = indexBy(
+          await PageObjects.settings.getSavedObjectElementsInTable(),
+          'title'
+        );
+        await elements['logstash-*'].checkbox.click();
+        await PageObjects.settings.clickSavedObjectsDelete();
 
         // Then, import the objects
-        await PageObjects.settings.clickKibanaSavedObjects();
         await PageObjects.settings.importFile(
           path.join(__dirname, 'exports', '_import_objects_with_index_patterns.ndjson')
         );
         await PageObjects.settings.checkImportSucceeded();
         await PageObjects.settings.clickImportDone();
-        // Wait for all the saves to happen
-        await PageObjects.settings.waitUntilSavedObjectsTableIsNotLoading();
 
         const objects = await PageObjects.settings.getSavedObjectsInTable();
         const isSavedObjectImported = objects.includes('saved object imported with index pattern');
         expect(isSavedObjectImported).to.be(true);
       });
     });
 
-    describe('.json file', () => {
+    describe.skip('.json file', () => {
```

Comment:
Lee, could you add a comment on top explaining the reason?

Reply:
removed.  that was temporary for testing.